core/invoker/src/main/scala/org/apache/openwhisk/core/invoker/InvokerReactive.scala

```diff
@@ -195,9 +197,14 @@ class InvokerReactive(
   }
 
   /** Stores an activation in the database. */
-  private val store = (tid: TransactionId, activation: WhiskActivation, context: UserContext) => {
-    implicit val transid: TransactionId = tid
-    activationStore.storeAfterCheck(activation, context)(tid, notifier = None)
+  private val store = if (activationStorageConfig.activationStoreEnabled) {
+    (tid: TransactionId, activation: WhiskActivation, context: UserContext) =>
+      {
```

Comment:
need the `{}` here?

Reply:
Technically not as its a single statement but it help in clarifying the boundaries

Reply:
I'm getting a little confused, if we set `activationStoreEnabled` to `false`, the `store` will do nothing here(code will not jump to `KafkaActivationStore.store` neither), then how can `activations` be sent to kafka? I see that only blocking activations result will be sent to `completedN` topic, but for non-blocking activations, they will be missing

Reply:
Support for non-blocking activations is currently pending.

Reply:
Is there any update on whether this will support non-blocking activations?